crates/ott-common/src/discovery/dns.rs

```diff
@@ -20,10 +20,13 @@ pub struct DnsServiceDiscoverer {
 
 impl DnsServiceDiscoverer {
     pub fn new(config: DnsDiscoveryConfig) -> Self {
-        info!(
-            "Creating DnsServiceDiscoverer, DNS server: {:?}",
-            config.dns_server
-        );
+        match config.dns_server {
+            None => info!(
+                "Creating DnsServiceDiscoverer, DNS server: {:?}",
+                config.dns_server
+            ),
+            Some(server) => (),
+        }
```

Comment:
This just makes it so that whenever `config.dns_server` is `Some`, this log will never get printed.

I've elaborated a bit more in the issue to make it more clear what the task is.